automatic/tv-browser/tools/chocolateyInstall.ps1

```diff
@@ -5,12 +5,20 @@ $toolsPath = Split-Path -parent $MyInvocation.MyCommand.Definition
 $packageArgs = @{
   packageName    = $env:ChocolateyPackageName
   fileType       = 'exe'
-  softwareName   = 'TV-Browser*'
-  file           = "$toolsPath\tvbrowser_4.0.1_win32.exe"
+  file           = gi $toolsPath\*.exe
   silentArgs     = '/S'
   validExitCodes = @(0)
+  softwareName   = 'TV-Browser*'  
 }
-
 Install-ChocolateyInstallPackage @packageArgs
-
 Get-ChildItem $toolsPath\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { Set-Content "$_.ignore" } }
+
+$jreDir = gi $Env:ProgramFiles\AdoptOpenJDK\jdk-11*-jre | select -First 1
+if (!$jreDir) { Write-Warning "Couldn't find AdoptOpenJDK JRE folder"; return }
```

Comment:
Shouldn't we test for the adoptopenjdk directory before installing the application? And then throw an exception if it doesn't exist?

Reply:
I am testing (`!$jreDIr`). 
I opted to let the app bee even without this as a) it still might work depending on JRE you already have b) its relativelly easy to fix by hand if needed. I could throw there too..

Reply:
All of this is removed in latest commit due to newer JRE dependency setting it correctly.